diff --git a/node_modules/@nuxt/webpack/dist/webpack.js b/node_modules/@nuxt/webpack/dist/webpack.js
index 5b2d005..a269de4 100644
--- a/node_modules/@nuxt/webpack/dist/webpack.js
+++ b/node_modules/@nuxt/webpack/dist/webpack.js
@@ -6,6 +6,10 @@
 */
 'use strict';
 
+const crypto = require("crypto");
+const crypto_orig_createHash = crypto.createHash;
+crypto.createHash = algorithm => crypto_orig_createHash(algorithm == "md4" ? "sha256" : algorithm);
+
 Object.defineProperty(exports, '__esModule', { value: true });
 
 const path = require('path');
